app/views/layouts/application.html.haml

```diff
@@ -66,10 +66,15 @@
               %div.govuk-grid-column-one-third
                 %h2.govuk-heading-m
                   = t('footer.for_job_seekers')
-                %ul.govuk-footer__inline-list
-                  %li.govuk-footer__inline-list-item
+                %ul.govuk-footer__inline-list.hide-bullet-points
+                  %li.govuk-footer-list-item
                     =link_to t('footer.search_teaching_vacancies'), root_url, class: 'govuk-footer__link'
-
+                  - if JobseekerAccountsFeature.enabled? && !authenticated?
```

Comment:
I think Christian mentioned this earlier, but perhaps it might be a good idea to rename the hiring staff helper method to something like `hiring_staff_signed_in?`?

Reply:
I started a little search-and-replace operation but was going to ask around for a name and wait for this to be merged before 😅